### DISCUSSION TITLE
Rendere wifi telecomando apertura/chiusura pozzo

### DISCUSSION THREAD

**Discussion by monteguly** [indomus/forum#3123] [GENERALE - Discussioni generiche]
Ho un telecomando piuttosto datato, utile per aprire a distanza un pozzo per irrigazione. Le funzioni sono accessibili tramite 2 tasti:apertura e chiusura. Alimentazione tramite batteria. Vorrei renderlo utilizzabile da remoto, tramite WiFi. Cosa posso fare ed acquistare? Grazie a chiunque vorrà aiutarmi. 
![IMG_20240616_111413](https://github.com/indomus/forum/assets/172965681/bf599a98-4313-4d04-a15a-e6dbbcb84f16)
![IMG_20240616_111351](https://github.com/indomus/forum/assets/172965681/2bdbb56f-2854-478b-b8d5-2b0a98afd1a0)

**Comment by indomus** [comment]
se è rolling puoi fare [così](https://indomus.it/progetti/integrare-in-domotica-qualsiasi-telecomando-anche-rolling-code-tramite-contatto-pulito/), altrimenti ti basta usare un [rm4 pro](https://indomus.it/componenti/broadlink-rm-pro-4/).

**Reply by monteguly** [reply]
Grazie mille👍
